bootstrap.php

```diff
@@ -12,7 +12,11 @@
     define('LARAVEL_START', microtime(true));
 }
 
-if (file_exists($applicationPath = getcwd().'/bootstrap/app.php')) { // Applications and Local Dev
+if (false !== $applicationPath = $_ENV['LARAVEL_BOOTSTRAP_FILE_PATH'] ?? $_SERVER['LARAVEL_BOOTSTRAP_FILE_PATH'] ?? false) { // Custom path in environment
```

Comment:
```suggestion
if (false !== $applicationPath = getenv('LARAVEL_BOOTSTRAP_FILE_PATH')) { // Custom path in environment
```

Reply:
We don't want to check `$_SERVER`?

Reply:
After a second thought, you are absolutely right – we don't need that. 🤝